ServerCore/Pages/EventSpecific/FAQ.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿@page "/{eventId}/{eventRole}/faq"
+@model ServerCore.Pages.EventSpecific.FAQModel
+@{
+    ViewData["Title"] = "FAQ";
+    ViewData["AdminRoute"] = "/EventSpecific/FAQ";
```

Comment:
I don't understand why these use different route info but maybe it will become clear when I get to the page that uses them.

Reply:
I need to set all three routes so that no matter which role the user selects I'll have a non-default value for that role. It just so happens that for this page they're all the same :)